logback-access/src/main/java/ch/qos/logback/access/spi/AccessEvent.java

```diff
@@ -50,14 +51,17 @@
   private transient final HttpServletRequest httpRequest;
   private transient final HttpServletResponse httpResponse;
 
+  String queryString;
   String requestURI;
   String requestURL;
   String remoteHost;
   String remoteUser;
   String remoteAddr;
+  String threadName;
   String protocol;
   String method;
   String serverName;
+  String sessionID;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/tony19/logback/file/2371246231/issues/source?bid=3222618&fileBranchId=3222701#l64)